ocpp/routing.py

```diff
@@ -56,7 +56,7 @@ def inner(*args, **kwargs):
     return decorator
 
 
-def after(action):
+def after(action, inject_response=False):
```

Comment:
can you include notes in the doc strings in respect to the new argument and its purpose?